plugins/modules/docker_container.py

```diff
@@ -2403,6 +2419,19 @@ def has_different_configuration(self, image):
                         # expected_healthcheck comparison in this case.
                         continue
 
+                    if key == 'expected_labels' and self.parameters.image_label_mismatch == 'fail':
+                        # If there are labels from the base image that should be removed and
+                        # base_image_mismatch is fail we want raise an error.
+                        image_labels = self._get_image_labels(image)
+                        would_remove_labels = []
+                        for label in image_labels.keys():
+                            if label not in self.parameters.labels:
+                                # Format label for error message
+                                would_remove_labels.append(repr(label))
```

Comment:
```suggestion
                                would_remove_labels.append(label)
```

Reply:
I used `repr` to quote the labels here, since they can contain spaces which would in my opinion make the error message confusing e.g. `Labels: foo bar, qux` instead of `Labels: 'foo bar', 'qux'`. Do you think I can ignore the quotes or maybe just make it more explicit like `"{}".format(label)`